client/src/components/Nav.js

```diff
@@ -46,8 +46,6 @@ class Nav extends Component {
 			<SubNav
 				componentClass={Scrollspy}
 				className="nav"
-				items={['info', 'laydown', 'approvals', 'poams', 'reports']}
```

Comment:
just out of curiosity, were these interfering with the tests or are they simply not needed?

Reply:
It's an undefined attribute (making the test fail under IE), and also not needed.